### PR TITLE
Added unit tests in pkg/es/config

### PR DESCRIPTION
**Pull request by Nabil-Salah**
## Which problem is this PR solving?
Part of: #5068 

## Description of the changes
- This commit adds tests for the `pkg/es/config` package.

## How was this change tested?
- `make test`

## Checklist
- [x] I have read https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md
- [x] I have signed all commits
- [x] I have added unit tests for the new functionality
- [x] I have run lint and test steps successfully
  - for `jaeger`: `make lint test`
  - for `jaeger-ui`: `yarn lint` and `yarn test`

### CODE REVIEW

**Review by yurishkuro — COMMENTED** [review]
you need to add a goleak test, like this: https://github.com/jaegertracing/jaeger/blob/4220d62cf3e2f18356c8504666ff43a1d838c241/pkg/version/package_test.go

**Comment by yurishkuro** [comment]
Nit: avoid using description like `partially fixes: #5068` - github reads it as `fixes #5068` and closes the corresponding issue. Instead you can say "Part of: #5068`

File: pkg/es/config/config_test.go [hunk 1]
```diff
@@ -0,0 +1,629 @@
...
+			if test.expectedError {
+				require.Error(t, got)
+			} else {
+				require.NoError(t, got)
+			}
+		})
+	}
+}
+func TestMain(m *testing.M) {
+	goleak.VerifyTestMain(m)
```
Comment (yurishkuro) [inline pkg/es/config/config_test.go:628]: not correct - see https://github.com/jaegertracing/jaeger/blob/4220d62cf3e2f18356c8504666ff43a1d838c241/pkg/version/package_test.go

**Comment by yurishkuro** [comment]
looks like your new tests are leaking goroutines. Make sure you always close ES clients properly.

**Comment by Nabil-Salah** [comment]
> looks like your new tests are leaking goroutines. Make sure you always close ES clients properly.

done now 
sorry didn't close config.TLS D:
but it's closed and all running play now

**Comment by yurishkuro** [comment]
Thanks!

**Pull request merged**